packages/flame_test/lib/src/test_flame_game.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flame/game.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:meta/meta.dart';
+
+/// Utility function for writing tests that require a [FlameGame] instance.
+///
+/// This function creates a [FlameGame] object, properly initializes it, then
+/// passes on to the user-provided test [testBody], and in the end disposes of
+/// the game object.
+///
+/// Example of usage:
+/// ```dart
+/// testFlameGame(
+///   'MyComponent can be added to a game',
+///   (game) async {
+///     final component = MyComponent()..addToParent(game);
+///     await game.ready();
+///     expect(component.isMounted, true);
+///   },
+/// );
+/// ```
+///
+/// The `game` instance supplied by this function to your test [testBody] is a
+/// standard [FlameGame]. If you want to have any other game instance, use the
+/// [testUserGame] function.
+@isTest
+Future<void> testFlameGame(
```

Comment:
We should add the equivalent of these methods to the game widget test method, like we have on the Flame Tester class, wdyt?

Reply:
Probably, though I'm not very familiar with how the widget tests work.

Reply:
you could base on the widgetTest on the FlameTester class.

But we could also leave that to a follow up too